### PR TITLE
fix: #tno-2564 - switched from EmailTo being a string array

### PR DESCRIPTION
**Pull request by kylgarmor**
- switch ed to just a plain string which should be in CSV format
- array is parsed where neccesary
- switch all YAML from using an array index for the SERVICE_MAILTO value

### CODE REVIEW

**Review by Fosol — APPROVED** [review]
You'll need to manually update the various DeployConfig and ConfigMap in DEV and TEST.

**Comment by kylgarmor** [comment]
> You'll need to manually update the various DeployConfig and ConfigMap in DEV and TEST.

Yup, I'm going to rope you in to help ;)

File: openshift/kustomize/services/capture/base/deploy.yaml [hunk 1]
```diff
@@ -133,17 +133,17 @@ spec:
             - name: CHES__From
               valueFrom:
                 configMapKeyRef:
-                  name: capture-service
+                  name: transcription-service
```
Comment (Fosol) [inline openshift/kustomize/services/capture/base/deploy.yaml:136]: capture?

File: openshift/kustomize/services/clip/base/deploy.yaml [hunk 2]
```diff
@@ -133,17 +133,17 @@ spec:
             - name: CHES__From
               valueFrom:
                 configMapKeyRef:
-                  name: clip-service
+                  name: transcription-service
```
Comment (Fosol) [inline openshift/kustomize/services/clip/base/deploy.yaml:136]: clip?

File: openshift/kustomize/services/content/base/deploy.yaml [hunk 3]
```diff
@@ -134,18 +134,36 @@ spec:
...
               valueFrom:
                 configMapKeyRef:
-                  name: content-service
+                  name: transcription-service
                   key: CHES_FROM
             - name: CHES__EmailEnabled
               valueFrom:
                 configMapKeyRef:
-                  name: content-service
+                  name: transcription-service
```
Comment (Fosol) [inline openshift/kustomize/services/content/base/deploy.yaml:142]: content?

File: openshift/kustomize/services/contentmigration/base/deploy.yaml [hunk 4]
```diff
@@ -137,17 +137,17 @@ spec:
             - name: CHES__From
               valueFrom:
                 configMapKeyRef:
-                  name: contentmigration-service
+                  name: transcription-service
```
Comment (Fosol) [inline openshift/kustomize/services/contentmigration/base/deploy.yaml:140]: contentmigration?

File: openshift/kustomize/services/filecopy/base/deploy.yaml [hunk 5]
```diff
@@ -150,17 +150,17 @@ spec:
             - name: CHES__From
               valueFrom:
                 configMapKeyRef:
-                  name: filecopy-service
+                  name: transcription-service
```
Comment (Fosol) [inline openshift/kustomize/services/filecopy/base/deploy.yaml:153]: filecopy?

File: openshift/kustomize/services/indexing/base/deploy.yaml [hunk 6]
```diff
@@ -170,17 +170,17 @@ spec:
             - name: CHES__From
               valueFrom:
                 configMapKeyRef:
-                  name: indexing-service
+                  name: transcription-service
```
Comment (Fosol) [inline openshift/kustomize/services/indexing/base/deploy.yaml:173]: indexing?

File: openshift/kustomize/services/syndication/base/deploy.yaml [hunk 7]
```diff
@@ -116,22 +116,27 @@ spec:
...
+                configMapKeyRef:
+                  name: syndication-service
+                  key: SEND_EMAIL_ON_FAILURE
 
             # CHES Configuration
             - name: CHES__From
               valueFrom:
                 configMapKeyRef:
-                  name: syndication-service
+                  name: transcription-service
```
Comment (Fosol) [inline openshift/kustomize/services/syndication/base/deploy.yaml:129]: syndication?

**Review by Fosol — CHANGES_REQUESTED** [review]

**Review by Fosol — APPROVED** [review]

File: openshift/kustomize/services/content/base/deploy.yaml [hunk 8]
```diff
@@ -146,6 +146,23 @@ spec:
                 configMapKeyRef:
                   name: content-service
                   key: CHES_EMAIL_AUTHORIZED
+            - name: Service__UnpublishedIndex
```
Comment (Fosol) [inline openshift/kustomize/services/content/base/deploy.yaml:149]: Thanks for fixing this

File: openshift/kustomize/shared_resources/base/pvc.yaml [hunk 9]
```diff
@@ -1,45 +1,45 @@
...
-    component: ingest-storage
-    managed-by: kustomize
-spec:
-  storageClassName: netapp-file-backup
-  accessModes:
-    - ReadWriteMany
-  resources:
-    requests:
-      storage: 5Gi
+# ---
```
Comment (Fosol) [inline openshift/kustomize/shared_resources/base/pvc.yaml:1]: Is this removed for a reason?  Don't we need this for the various Ingest Services?

**Review by Fosol — APPROVED** [review]
Much easier to maintain

**Pull request merged**